entgo-sveltekit/go.mod

```diff
@@ -0,0 +1,56 @@
+module app
```

Comment:
This one also needs to be added to .github/dependabot.yml

Reply:
I made the fix.